fms_core/models/_validators.py

```diff
@@ -11,4 +11,4 @@
 # one character less than that, since when renaming containers we need to append a
 # temporary character to prevent integrity errors.
 name_validator = RegexValidator(re.compile(r"^[a-zA-Z0-9.\-_]{1,199}$"))
-container_barcode_validator = RegexValidator(re.compile(r"^[^\\" + TEMPORARY_RENAME_SUFFIX + "]{1,199}$"))
+container_barcode_validator = RegexValidator(re.compile("^[^\\" + TEMPORARY_RENAME_SUFFIX + "]{1,199}$"))
```

Comment:
the r in front here lets you not have to escape the backslash with another, which makes it a bit more readable

Reply:
I had a problem with the \ actually escaping the " if not escaped and once escaped it read as if it was an escaped \ i wanted instead of an escaped $. There may be some more elegant way of doing this...